src/const.js

```diff
@@ -5,6 +5,7 @@ export const ESLINT_WARNING = 1;
 // and unwrappers (Sanitizer.unwrapSafeHTML, unwrapSafeHTML)
 // which are allowed by default by this plugin.
 const DISABLE_ALLOWED_ESCAPERS = {
```

Comment:
Nit, we may rename this const into NO_UNSANITIZED_OPTIONS or something like that, given that it isn't only about escapers allowed anymore. Given that this is a temporary measure, I don't consider that as a blocking issue if you don't.

Reply:
Good idea.